src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -420,6 +440,36 @@ public async Task<uint256> Rewind()
             }
         }
 
+        private void RestoreOutputs(RewindData rewindData)
```

Comment:
We need entry and exit logs for both of these new methods

Reply:
done